packages/itwinui-react/src/core/Tabs/Tabs.tsx

```diff
@@ -14,24 +14,31 @@ import {
 import '@itwin/itwinui-css/css/tabs.css';
 import { Tab } from './Tab';
 
-type TabsOrientationProps =
+type TabsTypeProps =
   | {
+      /**
+       * Type of the tabs.
+       *
+       * If `orientation = 'vertical'`, `pill` is not applicable.
+       * @default 'default'
+       */
+      type?: 'default' | 'borderless';
       /**
        * Orientation of the tabs.
        * @default 'horizontal'
        */
-      orientation?: 'horizontal';
+      orientation?: 'horizontal' | 'vertical';
       /**
-       * Type of the tabs.
+       * Content displayed to the right/bottom of the horizontal/vertical tabs
        *
-       * If `orientation = 'vertical'`, `pill` is not applicable.
-       * @default 'default'
+       * If `type = 'pill'`, `actions` is not applicable.
        */
-      type?: 'default' | 'borderless' | 'pill';
+      actions?: React.ReactNode[];
     }
   | {
-      orientation: 'vertical';
-      type?: 'default' | 'borderless';
+      type: 'pill';
+      orientation: 'horizontal';
+      actions?: undefined;
```

Comment:
Why doesn't `actions` have the same type as above, `React.ReactNode[]`?

Reply:
Because `actions` is not supported for `type: 'pill'`. It is only supported for default and borderless tabs.

I also thought of not adding `actions` on L41, but that gave type errors inside `Tabs.tsx` itself and said that `actions` is not defined in `TabsProps`.

![image](https://user-images.githubusercontent.com/45748283/213811931-3ae00a87-b788-4f79-b6ef-660496b0c327.png)

So I added `actions` as a prop, but gave it the type of `undefined`.

Reply:
After https://github.com/iTwin/iTwinUI/pull/1002#discussion_r1084223719, now `actions` is not included in the type definition at all (not even as `actions?: undefined`). So my above comment of needing to include actions in the type as `actions?: undefined` is outdated.